newton/examples/basic/example_basic_plotting.py

```
@@ -87,16 +87,13 @@ def __init__(self, viewer, args):
-                with wp.ScopedCapture() as capture:
-                    self.simulate()
-                self.graph = capture.graph
-            except Exception as exc:
-                self.graph = None
-                warnings.warn(f"CUDA graph capture failed: {exc}", stacklevel=2)
-        else:
+        try:
+            with wp.ScopedCapture() as capture:
+                self.simulate()
+            self.graph = capture.graph
+        except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)